homeassistant/components/steam_online/sensor.py

```diff
@@ -148,27 +156,22 @@ def _get_current_game(self):
         return repr(game_id)
 
     def _get_last_online(self):
-        last_online = datetime.fromtimestamp(
-            mktime(self._profile.last_online)
-        ).isoformat()
+        """Convert last_online from the steam module into timestamp UTC."""
+        last_online = utc_from_timestamp(mktime(self._profile.last_online))
 
         if last_online:
             return last_online
 
-    def _get_level(self):
-        level = self._profile.level
-
-        if level:
-            return level
+        return None
 
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
-        return (
-            {"game": self._game, "last_online": self._last_online, "level": self._level}
-            if self._game or self._last_online or self._level
-            else None
-        )
+        return {
+            "game": self._game if self._game else None,
```

Comment:
Our standard is to not add the item to the attributes at all if the value is `None`.

Reply:
Changed to reflect the standard